### DISCUSSION TITLE
Public Access

### DISCUSSION THREAD

**Discussion by stephenpapierski** [mealie-recipes/mealie#3745] [General]
It used to be that anyone visiting the site could view recipes, and you only had to login if you wanted to add, edit, etc. Is that still possible in the latest version?

I've followed the instructions here (https://docs.mealie.io/documentation/getting-started/usage/permissions-and-public-access/)
- [x] Group != "Private Group" 
- [x] Group = "Allow users outside of your group to see your recipes"
- [x] Recipe = "Public Recipe"

But I still get directed to the login page when navigate to the website. I'd like to be directed to all the public recipes, with the option to login if I need to change anything.

**Comment by stephenpapierski** [comment]
Ok, just discovered I can navigate to `example.com/g/home` and view everything as expected without having to be logged in. Is there a way I can have `example.com` go to that location instead of redirecting to `example.com/login`?

**Reply by michael-genson** [reply]
If you're on V1.2 or later this should be automatic, as long as your default group is `home` (which it should be, unless you changed it).

https://github.com/mealie-recipes/mealie/pull/2772

**Reply by stephenpapierski** [reply]
I'm on 1.8. My group name was 'Home', changing it to the lower case version seems to have fixed the behavior for me. Thanks!

**Reply by michael-genson** [reply]
Hmm, can you try switching it back to "Home" and see if it still works? Case shouldn't matter, but it could be a bug

**Reply by stephenpapierski** [reply]
Case seems to matter. Switching back to Home reverted to directing me to the sign in page.

**Reply by michael-genson** [reply]
Okay, I'll take a look. Thanks for letting me know!
